packages/features/ee/teams/components/MemberInvitationModal.tsx

```diff
@@ -271,24 +269,22 @@ export default function MemberInvitationModal(props: MemberInvitationModalProps)
                 </Button>
               )}
             </div>
-            <Button
-              type="button"
-              color="minimal"
-              onClick={() => {
-                props.onExit();
-                newMemberFormMethods.reset();
-              }}>
-              {t("cancel")}
-            </Button>
-            <Button
-              type="submit"
-              color="primary"
-              className="ms-2 me-2"
-              data-testid="invite-new-member-button">
-              {t("send_invite")}
-            </Button>
-          </DialogFooter>
+          </div>
         </Form>
```

Comment:
This is the issue. Then submit buttons aren't inside the form anymore so the form never gets submitted.

Reply:
This is breaking functionality BTW.

Reply:
Pushed a fix. The form still looks fine.

<img width="597" alt="image" src="https://github.com/calcom/cal.com/assets/3504472/c71406b5-2c21-496e-9e38-08edd3eb0286">

Waiting for e2e tests to pass again before merging.

Reply:
yes perfect. i had merged a broken merge conflict